lochness/tree/__init__.py

```diff
@@ -74,18 +76,37 @@ def get(data_type, base, **kwargs):
     raw_folder = None
     processed_folder = None
 
-    if 'raw' in Templates[data_type]:
-        raw_folder = Templates[data_type]['raw'].substitute(base=base, **kwargs)
+    if kwargs.get('BIDS', True):   # PHOENIX to BIDS
+        phoenix_id = Path(base).parent.name   # get SUBJECT
+        if 'raw' in Templates[data_type]:
+            # restructure root
+            base = Path(base).parent.parent / 'raw' / phoenix_id
+            raw_folder = Templates[data_type]['raw'].substitute(
+                    base=str(base), **kwargs)
+            raw_folder = Path(raw_folder).parent  # remove the 'raw' at the end
```

Comment:
These lines replace the folder paths, so the new data is arranged in the BIDS format, under the `PHOENIX/*/STUDY` directory. `metadata.csv` stays in the same location as before.